.storybook/fetch-mocks.js

```diff
@@ -43,7 +43,7 @@ export function fetchMockSaveSettings() {
 
 // Catch-all mock to log any other requests.
 export function fetchMockCatchAll() {
-	fetchMock.mock( '*', ( url, opts ) => {
+	fetchMock.catch( ( url, opts ) => {
```

Comment:
This is the correct way to implement a catch-all fetch mock. The old mock which used a wildcard prevented other mocks from being added since it would match everything instead of anything which wasn't matched 😄

Reply:
Thanks for the catch!